internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -58,6 +58,7 @@ const (
 	AgentPlatformStateInfo            = "AgentPlatformStateInfo"
 	adminPasswordSecretStringTemplate = "%s-admin-password"
 	adminKubeConfigStringTemplate     = "%s-admin-kubeconfig"
+	InstallConfigOverrides            = "hive.openshift.io/install-config-overrides"
```

Comment:
We should not use hive's name here unless it's going to be a hive feature. Is there some agreement that hive wants to implement this generally?

Reply:
Because Hive already have a way to set install config it should probably relate to assited controllers.
The controller api name is not decide yet, so we can use `adi.io.my.domain` for now.